tests/frontend/test_io.py

```diff
@@ -87,9 +87,9 @@ def prog():
 Dgate(0.735934779718964, 0.7469555733762603) | 1
 S2gate(0.543, -0.12) | [0, 3]
 Interferometer(A0) | [0, 1, 2, 3]
-MeasureHomodyne(phi=0) | 0
-MeasureHomodyne(select=0.32, phi=0.43) | 2
-MeasureHomodyne(select=0.32, phi=0.43) | 2
+MeasureHomodyne(0) | 0
+MeasureHomodyne(0.43, select=0.32) | 2
+MeasureHomodyne(0.43, select=0.32) | 2
```

Comment:
Changing this back, on the other hand (cf. https://github.com/XanaduAI/strawberryfields/pull/622/files#r698481744), would not work.